docs/DESIGN.md

```diff
@@ -0,0 +1,72 @@
+# Design
+
+* For CSS-in-JS we use styled components if either the styles are more than 2, supposed to be reused at multiple places, need responsive components (using media-queries).
+* rem and pixels both are acceptable.
+* We use [material-ui](https://material-ui.com/) library component and follow [Material Design](https://material.io/design/introduction/#principles) Guidelines
+
+## Colors and Fonts
+
+### Component Colors of Light theme
+
+
+* Application Background Colour: ![#ffffff](https://placehold.it/15/ffffff/000000?text=+) `#ffffff`
```

Comment:
Are these colors uopdated? Not sure, if we need them even.

@mariobehling Your thoughts?

Reply:
It can be added, but please get rid of images on external websites. Include all images in the /docs/images. Thanks